securedrop_client/app.py

```diff
@@ -81,18 +81,15 @@ def configure_logging(sdc_home: str) -> None:
     """
     All logging related settings are set up by this function.
     """
-    safe_mkdir(sdc_home, 'logs')
-    log_file = os.path.join(sdc_home, 'logs', 'client.log')
 
     # set logging format
     log_fmt = ('%(asctime)s - %(name)s:%(lineno)d(%(funcName)s) '
                '%(levelname)s: %(message)s')
     formatter = logging.Formatter(log_fmt)
 
-    # define log handlers such as for rotating log files
-    handler = TimedRotatingFileHandler(log_file, when='midnight',
-                                       backupCount=5, delay=False,
-                                       encoding=ENCODING)
+    # define log handlers using system's rsyslog
+    handler = SysLogHandler(address="/dev/log")
```

Comment:
we'll also want to add a check for the platform here as we do in `prevent_second_instance`, since this won't work on non-Linux platforms (recall that right now this project can be developed by contributors on other platforms, this is something we want to preserve)